odoo/addons/base/views/ir_actions_views.xml

```diff
@@ -345,7 +345,7 @@
                             <field name="update_path" widget="DynamicModelFieldSelectorChar" class="oe_inline" options="{'model': 'model_name'}"/>
                             <field name="update_field_id" invisible="1"/>
                             <field name="value_field_to_show" invisible="1"/>
-                            <field name="update_related_model_id" invisible="1"/>
+                            <field name="update_related_model_id" invisible="1" force_save="1"/>
```

Comment:
I swear i did this and it not work, maybe it works now lol, thanks anyway

Reply:
Hi, @duong77476-viindoo , take a look at Odoo Test Suite below or attached video. 
[odoo _test_entry.webm](https://github.com/user-attachments/assets/a2976280-2fcd-4dea-ab76-4cfe8dc608cc)